Configuration.props

```diff
@@ -54,12 +54,19 @@
     <HostOS Condition="$([MSBuild]::IsOSPlatform('linux'))">Linux</HostOS>
     <HostOS Condition="$([MSBuild]::IsOSPlatform('osx'))">Darwin</HostOS>
   </PropertyGroup>
+  <PropertyGroup>
+    <MicrosoftAndroidSdkPackName Condition="$([MSBuild]::IsOSPlatform('windows'))">Microsoft.Android.Sdk.Windows</MicrosoftAndroidSdkPackName>
+    <MicrosoftAndroidSdkPackName Condition="$([MSBuild]::IsOSPlatform('linux'))">Microsoft.Android.Sdk.Linux</MicrosoftAndroidSdkPackName>
+    <MicrosoftAndroidSdkPackName Condition="$([MSBuild]::IsOSPlatform('osx'))">Microsoft.Android.Sdk.Darwin</MicrosoftAndroidSdkPackName>
```

Comment:
Could this be simplified to?
```xml
<MicrosoftAndroidSdkPackName>Microsoft.Android.Sdk.$(HostOS)</MicrosoftAndroidSdkPackName>
```

Reply:
I added a new property because we override `$(HostOS)` when [packing on macOS](https://github.com/xamarin/xamarin-android/blob/main/build-tools/create-packs/Directory.Build.targets#L81-L82), but both packs need to pull files from the single Microsoft.Android.Sdk.Darwin output folder.